### DISCUSSION TITLE
Reverse search doesn't return house numbers

### DISCUSSION THREAD

**Discussion by schrodingerscatz** [osm-search/Nominatim#2743] [General]
When reverse searching on my local instance with just the Illinois dump imported only street name is returned. TIGER data has been imported. It seems like the max rank only goes to 26 when reverse searching regardless of what coordinates I put in. Import went all the way thru to 30 and did not fail. Is this an issue with the reverse geocoder or something that I messed up that's preventing it from returning house numbers on a reverse query?

**Comment by mtmail** [comment]
Can you add an example address/coordinate that works on nominatim.openstreetmap.org but doesn't work in your installation?

What do these database queries return for you?

`select rank_address, count(*) from placex group by rank_address order by rank_address;`

`select count(*) from location_property_tiger ;`
